vizro-ai/changelog.d/20240104_173912_maximilian_schulz_fix_vizro_ai_field_reference.md

```diff
@@ -0,0 +1,48 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Highlights ✨
+
+- A bullet item for the Highlights ✨ category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+<!--
+### Removed
+
+- A bullet item for the Removed category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+<!--
+### Added
+
+- A bullet item for the Added category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+<!--
+### Changed
```

Comment:
Hmm.. I am confused. I see a changelog change for an old PR but no code changes here. Does that mean that to fix the bug the user reported today there was no code change required? But we just had to release the latest version of vizro-ai because the bug should be fixed by Antonys old PR?

Reply:
Yes, so the old PR from A actually fixed an issue that was posted today. For some time I was trying to fix it without realising that it was already fixed... a bit annoying.

So I updated the old changelog fragment for that purpose, but it does not really matter as we should do a release soon, at which point it gets aggregated into the changelog

Going to focus on the release next

Reply:
Ahhh yeah, I thought his old PR had some useful information 😄 Alright sounds good!